client/my-sites/domains/domain-management/list/all-domains.jsx

```diff
@@ -756,29 +806,29 @@ const getFilteredDomainsList = ( state, context ) => {
 			} );
 
 		default:
-			return domainsList.filter(
-				( domain ) => domain.type !== domainTypes.WPCOM && canManageSitesMap[ domain.blogId ]
```

Comment:
would removing `canManageSitesMap[ domain.blogId ]` mean that a user who is a member of a site but not an admin would be able to see a list of domains owned by other admins of the same site?

Reply:
No, the domains are now filtered on the backend - see here: D119262-code#change-cOHTK7Y8aug5.